python/sidecar/http.py

```diff
@@ -1,24 +1,13 @@
 from dotenv import load_dotenv
 from fastapi import FastAPI
-
-import sidecar
-from sidecar.typing import (
-    ChatRequest,
-    ChatResponse,
-    DeleteRequest,
-    DeleteStatusResponse,
-    IngestRequest,
-    IngestResponse,
-    IngestStatusResponse,
-    ReferenceUpdate,
-    RewriteRequest,
-    RewriteResponse,
-    SearchRequest,
-    SearchResponse,
-    TextCompletionRequest,
-    TextCompletionResponse,
-    UpdateStatusResponse,
-)
+from sidecar import chat, ingest, rewrite, search, storage
```

Comment:
@gjreda please review this file changes bc I've had some issues calling the API endpoints in my local env!

Commit with the change (and more details) can be found [here](https://github.com/refstudio/refstudio/pull/382/commits/41df52f4817174d59075200b22069e085af3b823).